src/hunspell/affentry.cxx

```diff
@@ -217,7 +217,7 @@ struct hentry* PfxEntry::checkword(const char* word,
   // and if there are enough chars in root word and added back strip chars
   // to meet the number of characters conditions, then test it
 
-  int tmpl = len - appnd.size(); // length of tmpword
+  int tmpl = len - int(appnd.size()); // length of tmpword
```

Comment:
Please see https://google.github.io/styleguide/cppguide.html#Casting for some thoughts on this. Reasoning in above guide is very convincing for me. I suggested this as something which would make me much more comfortable merging this change.